src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -487,7 +491,71 @@ private void updateBundleAndRecipientListWithResponses(
         bundle.put(question, responses);
         recipientList.put(question.getId(), recipients);
     }
-    
+
+    /**
+     * Removes instructors who are not displayed to students from the
+     * {@link FeedbackQuestionBundle}
+     * 
+     * @param bundle
+     *            {@link FeedbackQuestionsBundle} containing questions for the
+     *            session
+     */
+    private void filterOutHiddenInstructors(FeedbackSessionQuestionsBundle bundle) {
```

Comment:
@unyoungwax, let's pass in the extra parameter then. Might be slightly more messy than what we have here now, but it will be preferable if both the instructor and student's code flow in the same way.